src/primitive.h

```diff
@@ -818,6 +840,7 @@ namespace toit {
 #define _A_T_PWMResourceGroup(N, name)    MAKE_UNPACKING_MACRO(PWMResourceGroup, N, name)
 #define _A_T_RpcResourceGroup(N, name)    MAKE_UNPACKING_MACRO(RpcResourceGroup, N, name)
 #define _A_T_RMTResourceGroup(N, name)    MAKE_UNPACKING_MACRO(RMTResourceGroup, N, name)
+#define _A_T_PcntUnitResourceGroup(N, name)   MAKE_UNPACKING_MACRO(PcntUnitResourceGroup, N, name)
```

Comment:
Not sure this needs a 3 space separator.

Reply:
done.